### PR TITLE
feature/events

### PR DESCRIPTION
**Pull request by MeqdamAlqudah**
1- Handle user events
2- add props to each element

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,18 +1,37 @@
-import { React, Component } from 'react';
+import React from 'react';
 import CalculatorOperations from './Buttons/CalculatorOperations';
 import Numbers from './Buttons/Numbers';
 import SimpleOperations from './Buttons/SimpleOperations';
 import classes from './Calculator.module.css';
 import Input from './Buttons/Input';
+import calculate from './logic/calculate';
+// import calculate from './logic/calculate';
```
Comment (Ghiftee) [inline src/components/Calculator.js]: - [ ] Kindly remove commented code. This makes your codebase cleaner.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,18 +1,37 @@
...
+    const { calc } = this.state;
+    const buttonName = e.target.name;
+    const newCalc = calculate(calc, buttonName);
+    this.setState({ calc: newCalc, innerText: `${newCalc.next ? newCalc.next : ''} ${newCalc.operation ? newCalc.operation : ''} ${newCalc.total ? newCalc.total : ''}` });
+  }
+
   render() {
+    // const finallClaultaions = (total, operation, next) => {
+    //   calculate({ total, next, operation });
+    // };
```
Comment (Ghiftee) [inline src/components/Calculator.js]: - [ ] Kindly remove commented code. This makes your codebase cleaner.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽  @MeqdamAlqudah 

Good job working on the project so far! 
✔️ You used state and props
✔️ There is no linter error
However, there are some issues that you need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [ ] When I try to perform a calculation in your app, since the numbers are entered from the right hand side, it does not display correctly. For instance, in the image below, I want to do ``` 6 / 3 ``` but it is displayed as ``` 3 / 6 ```. Although the result is correct, I suggest you make the display correct for a better user experience.

Screenshot 2022-01-06 at 16 07 38

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @MeqdamAlqudah, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**